api/src/test/java/org/motechproject/nms/api/osgi/CallDetailsControllerBundleIT.java

```diff
@@ -62,6 +63,11 @@
     @Inject
     private ServiceUsageCapDataService serviceUsageCapDataService;
 
+    public CallDetailsControllerBundleIT() {
+        System.setProperty("org.motechproject.testing.osgi.http.numTries", "1");
+        System.setProperty("org.motechproject.testing.osgi.http.msWait", "1000");
+    }
```

Comment:
Ah crap, good point, this is a bad example use. Damn!

On Fri, Apr 24, 2015 at 2:50 PM, Rob LaRubbio <notifications@github.com>
wrote:

> In
> api/src/test/java/org/motechproject/nms/api/osgi/CallDetailsControllerBundleIT.java
> <https://github.com/koshalt/mim/pull/90#discussion_r29088767>:
>
> > @@ -62,6 +63,11 @@
> >      @Inject
> >      private ServiceUsageCapDataService serviceUsageCapDataService;
> >
> > +    public CallDetailsControllerBundleIT() {
> > +        System.setProperty("org.motechproject.testing.osgi.http.numTries", "1");
> > +        System.setProperty("org.motechproject.testing.osgi.http.msWait", "1000");
> > +    }
>
> If there is only one try is the wait property needed? :)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/koshalt/mim/pull/90/files#r29088767>.
>



-- 
Frank Huster
Mobile Health Innovations
Grameen Foundation
+1.206.325.6690 x201